static/include/services/apis/base-rc.md

```diff
@@ -0,0 +1,4 @@
+Method Name | Description
```

Comment:
With code comment above, are you intending to make the service client method names code format separately from attribute? fine w/ me either way but thought i'd clarify

Reply:
Naw I wasn't changing anything with that--I just copy pasted from what was already there. But you're right, it is inconsistent! Will make it consistent now.

Reply:
Right now on https://docs.viam.com/program/apis/ it's about 50/50 haha

Reply:
(Will address once Slack convo comes to a consensus)

Reply:
A consensus has been reached! The people have spoken!! 😄

Reply:
Let there be code! :D